install.sh

```diff
@@ -111,24 +111,29 @@ main() {
     elif [ "$os" = Darwin ]; then
         if available brew; then
             NONINTERACTIVE=1 show brew install --cask brave-browser
+            echo "Installation complete! Start Brave by typing: open -a Brave\ Browser"
         else
-            error "Could not find brew. Please install brew to proceed." ""\
-                "A Brave .dmg can also be downloaded from https://brave.com/download/"
+            if available curl || available wget; then
+                show $curl -L https://laptop-updates.brave.com/latest/osx >~/Downloads/Brave-Browser.dmg
```

Comment:
This could be vulnerable to a symlink attack. Maybe we should simply have the `else` clause here?

Reply:
We could do `~/Downloads/brave-XXXXX/Brave-Browser.dmg` (where `XXXXX` is generated by `mktemp -d` .